knowledge-base/chart-datatable-binding.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How to use DataTable as source of data for the Chart
```

Comment:
This title is long and uses keywords, which the customer didn't, so they may be unnecessary at least in the title ("use", "source"). At the same time, the customer used the keyword "bind", which we mention just once in the whole article in another grammar form ("binding"). The same applies for the word "convert" in the meta description - the customer is converting apps, not data sources.

Overall, our KBs must capture the customer context, which means to use the keywords used by customers. We can add more keywords, but not at the cost of the required ones.